src/java.net.http/share/classes/jdk/internal/net/http/Stream.java

```diff
@@ -673,7 +674,8 @@ request, exchange, responseHeaders, connection(),
             completeResponse(response);
         } else {
             if (Log.headers()) {
-                StringBuilder sb = new StringBuilder("TRAILING HEADERS:\n");
+                StringBuilder sb = new StringBuilder("TRAILING HEADERS (streamid=%d):\n".formatted(streamid));
+                sb.append("  %s %s %d\n".formatted(request.method(), request.uri(), responseCode));
```

Comment:
These are trailing headers - printing URI and response code would be confusing here. Only printing streamId should be fine.

```suggestion
```